package.json

```
@@ -30,8 +30,8 @@
     "build": "tsc --build --verbose tsconfig.build.json",
     "build:integrations": "pnpm run --filter \"@configu-integrations/*\" build",
     "test": "tsx --test \"packages/**/src/**/*.test.ts\" \"packages/integrations/expressions/**/src/**/*.test.ts\" \"packages/integrations/formatters/**/src/**/*.test.ts\" \"packages/integrations/stores/**/src/**/*.test.ts\"",
-    "test:cov": "tsx --test --experimental-test-coverage --test-reporter=spec --test-reporter=lcov --test-reporter-destination=stdout --test-reporter-destination=lcov.info packages/**/src/**/*.test.ts",
-    "test:watch": "tsx --test --watch packages/**/src/**/*.test.ts",
+    "test:cov": "tsx --test --experimental-test-coverage --test-reporter=spec --test-reporter=lcov --test-reporter-destination=stdout --test-reporter-destination=lcov.info  \"packages/**/src/**/*.test.ts\" \"packages/integrations/expressions/**/src/**/*.test.ts\" \"packages/integrations/formatters/**/src/**/*.test.ts\" \"packages/integrations/stores/**/src/**/*.test.ts\"",
+    "test:watch": "tsx --test --watch  \"packages/**/src/**/*.test.ts\" \"packages/integrations/expressions/**/src/**/*.test.ts\" \"packages/integrations/formatters/**/src/**/*.test.ts\" \"packages/integrations/stores/**/src/**/*.test.ts\"",
```

@davidmeirlevy I started reviewing this, and noticed you get just as much of a performance improvement by not specifying a path at all, and just sticking to `tsx`'s default - which might actually be better (if more packages are added, no need to keep updating these).
```suggestion
    "test": "tsx --test",
    "test:cov": "tsx --test --experimental-test-coverage --test-reporter=spec --test-reporter=lcov --test-reporter-destination=stdout --test-reporter-destination=lcov.info" \"packages/integrations/formatters/**/src/**/*.test.ts\" \"packages/integrations/stores/**/src/**/*.test.ts\"",
    "test:watch": "tsx --test --watch",
```

Might be worth considering this option too if there's no specific reason to override the default.

---

there's a reason.
the default values are not good for running internal monorepo packages without running the node_modules tests.